lab_4_retrieval_w_clustering/main.py

```diff
@@ -113,6 +148,15 @@ def put_corpus(self, corpus: Corpus) -> None:
                 or if input arguments are empty,
                 or if methods used return None.
         """
+        if not isinstance(corpus, list) or not corpus:
+            raise ValueError("Input arguments are wrong")
+        tokenized_documents = self._tokenizer.tokenize_documents(corpus)
+        bm = self._vectorizer
+        for paragraph in tokenized_documents:
+            new = self._tokenizer._remove_stop_words(paragraph)
+            if not new:
+                del new
```

Comment:
for what?

Reply:
where are you using ```new```?

Reply:
nowhere, I have to take them into account and not use them, I tried another way